File: SkillSync.WebServer/SkillSync.Domain/Models/Order.cs
using SkillSync.Domain.Enums;

namespace SkillSync.Domain.Models
{
    public class Order
    {
        public Guid Id { get; set; }

        public Guid ProjectId { get; set; }
        public Project Project { get; set; }

        public string CustomerId { get; set; }
        public User Customer { get; set; }

        public PackageType PackageType { get; set; }

        public OrderStatus Status { get; set; }

        public DateTime CreatedAt { get; set; }

        public DateTime UntilTo { get; set; }

        public DateTime? CompletedAt { get; set; }

        public List<OrderContent> Contents { get; set; }
    }
}

File: SkillSync.Client/src/app/shared/entities/models/order.ts
import { OrderStatus } from "../../enums/order-status";
import { PackageType } from "../../enums/packages";
import { OrderContent } from "./order-content";

export interface Order {
    id: string;
    projectId: string;
    projectTitle: string;
    customerName: string;
    customerEmail: string;
    freelancerName: string;
    freelancerId: string;
    price: number;
    packageType: PackageType;
    status: OrderStatus;
    createdAt: Date;
    untilTo: Date;
    completedAt?: Date;
    previewContents: OrderContent[];
    finalProductContents: OrderContent[];
}